ios/Video/RCTVideo.swift

```diff
@@ -1639,6 +1678,31 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
         }
     }
 
+    @objc
+    func enterPictureInPicture() {
+        if (_pip?.isInitialized ?? false) == false {
+            initPictureinPicture()
+            _playerViewController?.allowsPictureInPicturePlayback = true
+        }
+        _pip?.enterPictureInPicture()
+    }
+
+    @objc
+    func exitPictureInPicture() {
+        guard _pip?.isPictureInPictureActive ?? false else { return }
+
+        _pip?.exitPictureInPicture()
+        #if os(iOS)
```

Comment:
I think you should also do that on `if #available(tvOS 14.0, *) {` ?

Reply:
I'm missing all tvOS case handling, I'll fix that throughout, but I don't have a real TV device so it's hard to test.🥲